docker/webui/run_openqa.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# prepare environment
+rm -f /run/dbus/pid
+/usr/share/openqa/script/upgradedb --user geekotest --upgrade_database
+
+# run services
+dbus-daemon --system --fork
+su geekotest -c '/usr/share/openqa/script/openqa-scheduler &'
+su geekotest -c '/usr/share/openqa/script/openqa-websockets &'
+su geekotest -c '/usr/share/openqa/script/openqa gru -m production run &'
+rcapache2 start && su geekotest -c '/usr/share/openqa/script/openqa prefork -m production --proxy'
```

Comment:
Since `start_daemon` is from sysvinit-tools, isn't it going to be deprecated? Also I couldn't really find any documentation for it. I can run:

    start_daemon -u geekotest /usr/share/openqa/script/openqa-scheduler

but it won't return. Should I run it in the background? I assumed that using `start_daemon` will take care for me.